### ISSUE TITLE
Possible to add other language in search ? 

### ISSUE THREAD

**Issue by razeNFR** [g0ldyy/comet#106]
Hello,

I tried your addon and he work fine but im french and sometimes when i search for a serie in frech like Camping Paradis, Nouvelle école or Comedy Class par Eric et Ramzy or somethings like this, the addon search with the english title and not the french one, so the addon cant fetch the title because the title is in french on Jackett result.

Sorry my english is very bad, i hope you understand what i mean.

**Comment by razeNFR** [comment]
> U can try my [comet-uncached](https://github.com/Zaarrg/comet-uncached) repo with some (experimental) features. This has support in the advanced settings for different language search. Meaning it can search in english, german and italian for example. Any feedback is greatly appreciate so i can fix and identify issues faster.

Ok thank you, i will try that repo.

And you think its possible you do a compose file with install of traefik automated in it (like jackettio) ?

**Comment by ericvlog** [comment]
Can add chinese language for support? thanks

**Comment by Zaarrg** [comment]
> Can add chinese language for support? thanks

As far as i know chinese should be supported. U can select it in language preference and language search. Enhanced Language also looks for those language codes in titles:
```js
'chi': 'zh', 'zho': 'zh', 'zh': 'zh', 'chinese': 'zh' 
```

**Comment by ericvlog** [comment]
> > Can add chinese language for support? thanks
> 
> As far as i know chinese should be supported. U can select it in language preference and language search. Enhanced Language also looks for those language codes in titles:
> 
> ```js
> 'chi': 'zh', 'zho': 'zh', 'zh': 'zh', 'chinese': 'zh' 
> ```

ops, it default select all, then I have installed it.....hope select all is okay for usage..

thanks!

**Comment by ericvlog** [comment]
@Zaarrg I went to https://github.com/Zaarrg/comet-uncached and using compose way to deploy, but i saw the image file is same for the comet image: g0ldyy/comet

 izzit i need change on that to have a  (experimental) features. ?

**Comment by Zaarrg** [comment]
> @Zaarrg I went to https://github.com/Zaarrg/comet-uncached and using compose way to deploy, but i saw the image file is same for the comet image: g0ldyy/comet
> 
> izzit i need change on that to have a (experimental) features. ?

Yes you need use ghcr.io/zaarrg/comet-uncached:latest
```docker pull ghcr.io/zaarrg/comet-uncached:latest```

**Comment by ericvlog** [comment]
> > @Zaarrg I went to https://github.com/Zaarrg/comet-uncached and using compose way to deploy, but i saw the image file is same for the comet image: g0ldyy/comet
> > izzit i need change on that to have a (experimental) features. ?
> 
> Yes you need use ghcr.io/zaarrg/comet-uncached:latest `docker pull ghcr.io/zaarrg/comet-uncached:latest`

Okay, thanks so much for your help.

**Comment by ericvlog** [comment]
@Zaarrg hello, i have enable few language search, but logs only show it search by russian language.

![image](https://github.com/user-attachments/assets/540faea3-f97f-4f31-ad8c-aec1af787600)

```
2024-09-04 07:37:15 | 🌠 COMET | main.start_log - Title Match Check: True
2024-09-04 07:37:15 | 🌠 COMET | main.start_log - Custom Header HTML: False
2024-09-04 07:38:17 | 📰 INFO | stream.stream - No cache found for The Monkey King: Reborn with user configuration
2024-09-04 07:38:18 | 📰 INFO | stream.stream - **Titles gathered for searching {'ru': 'Путешествие на Запад: Реинкарнация Царя демонов', 'default': 'The Monkey King: Reborn'}**
2024-09-04 07:38:18 | 📰 INFO | stream.stream - Start of jackett search for The Monkey King: Reborn with indexers ['rutor', 'mejortorrent', 'rutracker.org', '52bt', 'bitsearch', 'eztv', 'the_pirate_bay', 'therarbg', 'bitru', 'therarbg']
2024-09-04 07:38:19 | 📰 INFO | general.get_torrentio - 14 torrents found for The Monkey King: Reborn with Torrentio
2024-09-04 07:38:21 | 📰 INFO | general.get_zilean - 0 torrents found for The Monkey King: Reborn using title Путешествие на Запад: Реинкарнация Царя демонов with Zilean
2024-09-04 07:38:22 | 📰 INFO | general.get_zilean - 1 torrents found for The Monkey King: Reborn using title The Monkey King: Reborn with Zilean
2024-09-04 07:39:20 | ⚠️ WARNING | general.fetch_jackett_results - Exception while fetching Jackett results for indexer mejortorrent: 
2024-09-04 07:39:20 | 📰 INFO | stream.stream - 24 torrents found for The Monkey King: Reborn with jackett, Zilean,
```

**Comment by ericvlog** [comment]
I test on few chinse movies it only use russian language to search, 
and the search result uncached is Infront of cache result, from configuration pages result order it  on last only Infront Unknown.

` | 📰 INFO | stream.stream - Titles gathered for searching {'ru': 'Боевая мутантка', 'default': 'Mutant Ghost Wargirl'}`

**Comment by ericvlog** [comment]
@Zaarrg if I remove Russians on Language Search, other which enable will no function at all. It just use default.

```
Titles gathered for searching {'default': 'Warriors of Future'
```

**Comment by Zaarrg** [comment]
> @Zaarrg if I remove Russians on Language Search, other which enable will no function at all. It just use default.
> 
> ```
> Titles gathered for searching {'default': 'Warriors of Future'
> ```

Okay so should be fixed now in the latest commit. 

Also when there is no alternative language found it will be not listed in "Titles gathered for searching" as it uses imdb to find other languages and for the The Monkey King: Reborn movie there is no Japanese language for example listed. You can see [here](https://www.imdb.com/title/tt14391088/releaseinfo/#akas) for example what languages are listed and that Japanese is missing.
[Also enabled issues forgot to do it, my bad]

**Comment by ericvlog** [comment]
> > @Zaarrg if I remove Russians on Language Search, other which enable will no function at all. It just use default.
> > ```
> > Titles gathered for searching {'default': 'Warriors of Future'
> > ```
> 
> Okay so should be fixed now in the latest commit.
> 
> Also when there is no alternative language found it will be not listed in "Titles gathered for searching" as it uses imdb to find other languages and for the The Monkey King: Reborn movie there is no Japanese language for example listed. You can see [here](https://www.imdb.com/title/tt14391088/releaseinfo/#akas) for example what languages are listed and that Japanese is missing. [Also enabled issues forgot to do it, my bad]

Yup, I test on few movies, it now working!! it show result from same movies but different name by country. it more result now.
if other issue i will open on comet-uncached there.

`stream.stream - Titles gathered for searching {'CN': '城市猎人', 'JP': 'シティーハンター', 'RU': 'Городской охотник', 'default': 'City Hunter'}`

Thanks appreciate.
